src/com/service/PersonService.java

```diff
@@ -0,0 +1,53 @@
+package com.service;
+
+import com.person.Person;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class PersonService implements IService{
+
+//    public static final String SHOW_PERSONS =
```

Comment:
Don't leave commented out code. Other developer which reads this code will not be sure whether this piece of code is important or not. Anyway if you have good reason to leave commented code like this you should explain why(in a comment above for example)